lib/PHPExif/Adapter/FFprobe.php

```diff
@@ -73,8 +74,14 @@ public function setToolPath(string $path): FFprobe
     public function getToolPath(): string
     {
         if ($this->toolPath === '') {
-            // Do not use "which": not available on sh
-            $path = exec('command -v ' . self::TOOL_NAME);
+            try {
+                // Do not use "which": not available on sh
+                $path = exec('command -v ' . self::TOOL_NAME);
+            } catch (ExecException) {
+                // @codeCoverageIgnoreStart
+                $path = false;
+                // @codeCoverageIgnoreEnd
+            }
```

Comment:
Same issue here as in the comment above.

Reply:
Fixed : [ccc6fe1](https://github.com/LycheeOrg/php-exif/pull/48/commits/ccc6fe1aa1abdd207371f75fbb241a10fdf2af4a)